Automattic/woocommerce-payments
# REST API documentation

#### Changes proposed in this Pull Request

1. Initial documentation for our API endpoints.
2. A new Github workflow to publish API docs to https://automattic.github.io/woocommerce-payments.

The documentation static site generation is based on [Slate](https://github.com/slatedocs/slate). The theming and flow are heavily inspired by [woocommerce-rest-api-docs](https://github.com/woocommerce/woocommerce-rest-api-docs).

![image](https://user-images.githubusercontent.com/1288616/157829669-a91bd501-7570-46b1-a17a-2ad3cc40c4f4.png)

Docs are added as markdown files in `docs/rest-api/source/includes/wp-api-v3`. This PR ships only with the `Orders` controller docs.

#### Testing instructions

* Check the documentation markdown files.
* Check the documentation site at https://automattic.github.io/woocommerce-payments.

-------------------

- [x] Covered with tests (does not apply)
